packages/spartan/extra/dcos-spartan-watchdog.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I can rewrite it in Python, but the Python equivalent is significantly longer, and a bit more complicated IMHO (you have to do the DNS resolution yourself, fcntl in Python isn't super intuitive as an API)